packages/browser/src/browser.ts

```diff
@@ -1,7 +1,8 @@
 import fetch from "cross-fetch";
 
-import { ILogtailLog, ILogtailOptions } from "@logtail/types";
+import {Context, ILogLevel, ILogtailLog, ILogtailOptions, StackContextHint} from "@logtail/types";
```

Comment:
Pretend I'm ESlint:
Missing space after `{` and before `}`

Reply:
Is `StackContextHint` used in this file?

Reply:
it's not, thx